alpha/veneer/semver/README.md

```diff
@@ -1,6 +1,6 @@
 ## Semver Veneer:
```

Comment:
```suggestion
## Semver Veneer
```

Reply:
Line 71 is a little confusing to me. Would something like the following change the intended meaning? 
With the veneer attribute `GenerateMajorChannels: true` (skipping the rendered bundle image output), the resulting major channels from the command are: